Doc/library/datetime.rst

```diff
@@ -1735,7 +1740,7 @@ day, and subject to adjustment via a :class:`tzinfo` object.
    * ``fold in [0, 1]``.
 
    If an argument outside those ranges is given, :exc:`ValueError` is raised. All
-   default to ``0`` except *tzinfo*, which defaults to :const:`None`.
```

Comment:
Why was this link removed?

Reply:
This remark is outdated; marking as resolved.